actionners/cilium/networkpolicy/networkpolicy.go

```diff
@@ -0,0 +1,314 @@
+package networkpolicy
+
+import (
+	"context"
+	"fmt"
+	v2 "github.com/cilium/cilium/pkg/k8s/apis/cilium.io/v2"
+	v1 "github.com/cilium/cilium/pkg/k8s/slim/k8s/apis/meta/v1"
+	"github.com/cilium/cilium/pkg/policy/api"
+	"net"
+
+	errorsv1 "k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	cilium "github.com/falco-talon/falco-talon/internal/cilium/client"
+
+	"github.com/falco-talon/falco-talon/internal/events"
+	kubernetes "github.com/falco-talon/falco-talon/internal/kubernetes/client"
+	"github.com/falco-talon/falco-talon/internal/rules"
+	"github.com/falco-talon/falco-talon/utils"
+)
+
+type Config struct {
+	AllowCIDR []string `mapstructure:"allow_cidr" validate:"omitempty"`
```

Comment:
Can you also add the possibility to allow by namespace, see what I did for calico.